Documentation/json-schema/distro-support-schema.json

```diff
@@ -2,51 +2,127 @@
     "$schema": "http://json-schema.org/draft-04/schema#",
     "type": "object",
     "properties": {
-      "DISTRO": {
+        "$schema": {

```

Comment:
I had to describe the schema field because I turned on turbo-enforcement of the schema using `additionalProperties: false` everywhere.